content/en/logs/guide/docker-logs-collection-troubleshooting-guide.md

```diff
@@ -72,6 +72,8 @@ This status means that logs are enabled but you haven't specified which containe
 
 ## Docker log collection from file issues
 
+Docker logs are collected from the log files on disk by default in agent 6.33.0/7.33.0+ so long as the log files on disk are accessible by the agent. `DD_LOGS_CONFIG_DOCKER_CONTAINER_USE_FILE` can be set to `false` to disable this behavior. Some containers may still be tailed from the Docker socket despite the default if the logs file path is not accessible. If the Agent ever collected logs from a given container via the docker socket, it will continue to do so (including after Agent restarts) in order to avoid sending duplicate logs. To force the Agent to collect logs from file for such a container, set `DD_LOGS_CONFIG_DOCKER_CONTAINER_FORCE_USE_FILE` to `true`. This may cause duplicate logs when the transition occurs.
```

Comment:
```suggestion
The Agent collects Docker logs from the log files on disk by default in versions 6.33.0/7.33.0+ so long as the log files on disk are accessible by the Agent. `DD_LOGS_CONFIG_DOCKER_CONTAINER_USE_FILE` can be set to `false` to disable this behavior.

The Agent tails containers from the Docker socket if it cannot access the logs file path. If the Agent ever collected logs from a particular container using the Docker socket, it will continue to do so (including after Agent restarts) in order to avoid sending duplicate logs. To force the Agent to collect logs from file, set `DD_LOGS_CONFIG_DOCKER_CONTAINER_FORCE_USE_FILE` to `true`. This setting may cause duplicate logs to appear in Datadog.
```